packages/shared/src/components/notifications/EnableNotification.tsx

```diff
@@ -7,21 +7,23 @@ import CloseButton from '../CloseButton';
 import { cloudinary } from '../../lib/image';
 import VIcon from '../icons/V';
 import { webappUrl } from '../../lib/constants';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+import { AnalyticsEvent } from '../../lib/analytics';
 
 export enum NotificationPromptSource {
-  NotificationList = 'notificationList',
-  NewComment = 'newComment',
-  NewSource = 'newSource',
+  NotificationsPage = 'notifications page',
+  NewComment = 'new comment',
+  CommunityPicks = 'community picks modal',
 }
 
 const DISMISS_BROWSER_PERMISSION = 'dismissBrowserPermission';
 
 type DismissBrowserPermissions = Record<NotificationPromptSource, boolean>;
 
 const DEFAULT_CACHE_VALUE: DismissBrowserPermissions = {
```

Comment:
A general comment that you should address separately, dismissing is not per prompt but cross app.
Meaning if I dismissed the widget I should never see it again.

Reply:
Cool. Let me refactor this one.